app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    return unless session[:user_id]
+
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
dùng này thay cho !.nil? nhé
```suggestion
    current_user.present?
```